app/screens/Explore.js

```diff
@@ -771,6 +771,7 @@ class Explore extends React.Component {
                     onRegionDidChange={this.onRegionDidChange}
                     regionDidChangeDebounceTime={10}
                     onPress={this.onPress}
+                    attributionPosition={{ 'bottom': 40, 'left': 8 }}
```

Comment:
Slight suggested change to align with mapbox logo
```suggestion
                    attributionPosition={{ 'bottom': 36, 'left': 4 }}
```

Is the menu overlap only on iOS? Haven't tested on iOS yet, and do not see any menu overlap on Android.

Reply:
I saw it on both iOS and the Android emulator. Also thought it was a bit hard to press the attribution when the menu is there.